src/makeRobotAccountant.js

```diff
@@ -6,7 +6,16 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const summing = (number1) => (number2) => {
```

Comment:
Interesting solution)  👍  I've never seen it before 🤔

Reply:
let's better put inside functions on new line to improve code readability
```suggestion
  const summing = (number1) => {
    return (number2) => {
```